habitat-lab/habitat/config/habitat/task/objectnav_waypoint_control.yaml

```diff
@@ -0,0 +1,33 @@
+defaults:
+  - task_config_base
+  - actions:
+    - velocity_stop
```

Comment:
Same for waypoints -- is stop implied beneath thresholded distance/turn values?

Reply:
Same applies here